src/OllamaSharp/MicrosoftAi/AbstractionMapper.cs

```diff
@@ -13,6 +13,13 @@ namespace OllamaSharp.MicrosoftAi;
 /// </summary>
 internal static class AbstractionMapper
 {
+	private static readonly AIJsonSchemaTransformCache _schemaTransformCache = new(new()
+	{
+		ConvertBooleanSchemas = true,
```

Comment:
Note that these options merely reflect the MEAI defaults before the recent changes to schema generation were made. I believe they are not strictly necessary for the case of Ollama, but this approach offers individual `IChatClient` implementations ability to pick and choose transformations per their requirements.

Reply:
Yeah, I thought about `RequireAllProperties = true` here and if this should really be the default when using with Microsoft.Extension.AI.

Reply:
The internal MEAI Ollama implementation doesn't use it and its integration tests seem to be passing just fine. If you're ok with getting rid of it I can do that.

Reply:
I just wonder if we can have optional parameters with that implementation and if not, if it matters.

Reply:
It makes optional parameters impossible, so if you think that creates problems I can remove it.

Reply:
Removed.

Reply:
I think this is better, thank you